<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>             Entry to ARMED state!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="programStart">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>programStart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="configure">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="storeConfig">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>storeConfig</string>
    </property>
   </widget>
   <widget class="QPushButton" name="armTheSystem">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>armTheSystem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doorOpened">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>doorOpened</string>
    </property>
   </widget>
   <widget class="QPushButton" name="userGaveCorrectPin">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>userGaveCorrectPin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="userGaveWrongPin">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>userGaveWrongPin</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
